reana_cluster/backends/kubernetes/k8s.py

```diff
@@ -83,7 +83,7 @@ def __init__(self,
         logging.debug('Creating a ReanaBackend object '
                       'for Kubernetes interaction.')
 
-        # Load Kubernetes cluster configuration. If reana-cluster.yaml
+        # Load Kubernetes cluster configuration. If reana-cluster-stable.yaml
```

Comment:
See `reana-cluster-minikube.yaml` and friends started in https://github.com/reanahub/reana/pull/167/files#r298332058